js/core/utils/size.js

```diff
@@ -135,6 +135,8 @@ export const parseHeight = function(value, container) {
         value = parseInt(value);
     } else if(value.indexOf('vh') > 0) {
         value = window.innerHeight / 100 * parseInt(value.replace('vh', ''));
+    } else if(element && value.indexOf('em') > 0) {
+        value = parseFloat(value.replace('em', '')) * parseFloat(window.getComputedStyle(element).fontSize);
```

Comment:
What about 'rem' units? The same bug: [[codepen](https://codepen.io/marker-dao/pen/BaOvWxR)].